tests/Discord/AuthorTest.php

```diff
@@ -30,4 +30,10 @@
 			'url'      => null,
 		]);
 	});
+
+	it('gravatar', function () {
```

Comment:
The idea with the it syntax is to make it human readable in the code and also the results. 

```suggestion
	it('should return a proper gravatar URL', function () {
```